lib/anyway/rails/loaders/yaml.rb

```diff
@@ -4,27 +4,14 @@ module Anyway
   module Rails
     module Loaders
       class YAML < Anyway::Loaders::YAML
```

Comment:
Why do we leave this loader; it should become unnecessary after we added `current_environment` support to the base one.

The `known_environments` feature should also be ported to the base loader.

Reply:
This loader contains some specific Rails logic such as `known_environments` and relative config path definition. If I place it to base loader it will become a little messy.

Reply:
Plus `Settings.future.unwrap_known_environments` setting make sense only in Rails apps in pure Ruby apps this behavior enable by default.

Reply:
> Settings.future.unwrap_known_environments setting make sense only in Rails apps

Why so? Other apps could also have a notion of known envs (we just don't support them out-of-the-box right now); for example, Hanami has the same `development`, `test` and `production` envs by default.